router/service.go

```diff
@@ -39,6 +40,8 @@ func ServiceHandlers(base *gin.RouterGroup) {
 			service.PUT("", perm.MustPlatformAdmin(), middleware.Payload(), api.UpdateService)
 			service.DELETE("", perm.MustPlatformAdmin(), api.DeleteService)
 
+			service.POST("/stream", api.PostServiceStream)
```

Comment:
I wonder if we should explore locking down this endpoint behind some permission layer?

Thinking for the other endpoints we call `perm.MustPlatformAdmin()` so perhaps we adopt that?

```suggestion
			service.POST("/stream", perm.MustPlatformAdmin(), api.PostServiceStream)
```

Reply:
👍🏼